packages/design-tokens/src/tokens/functional/components/logosuite/base.json

```diff
@@ -6,7 +6,12 @@
           "value": "var(--base-size-80)"
         },
         "rowGap": {
-          "value": "var(--base-size-40)"
+          "mobile": {
+            "value": "var(--base-size-40)"
+          },
+          "default": {
+            "value": "var(--base-size-80)"
+          }
```

Comment:
FYI, you don't need to create separate tokens for different viewports, as we have a custom token formatter that handles responsive behaviour on the same variable. [Example from river](https://github.com/primer/brand/blob/main/packages/design-tokens/src/tokens/functional/components/river/river.json#L7).

Reply:
Ah nice, thank you.

Is there something I need to do to enable it for a component?

I've updated my base.json to the following but I'm still not getting the responsive behaviour in the outputted `/lib/design-tokens/css/tokens/functional/components/logosuite/base.css` file

```json
{
  "brand": {
    "LogoSuite": {
      "logobar": {
        "columnGap": {
          "value": "var(--base-size-80)"
        },
        "rowGap": {
          "value": "var(--base-size-40)",
          "responsive": {
            "768px": {
              "value": "var(--base-size-80)"
            }
          }
        },
        "marquee": {
          "default": {
            "value": "30s"
          },
          "slow": {
            "value": "60s"
          },
          "gap": {
            "value": "var(--base-size-96)"
          }
        }
      }
    }
  }
}
```

Reply:
Yes, you have to add it to the [`build-tokens` script file](https://github.com/primer/brand/blob/main/packages/design-tokens/scripts/build-tokens.js#L161-L171).

Reply:
Amazing, thanks @danielguillan 💖